environment.yml

```diff
@@ -2,48 +2,57 @@ name: simpeg-test
 channels:
   - conda-forge
 dependencies:
+# dependencies
   - numpy>=1.20
   - scipy>=1.8
   - scikit-learn>=1.2
-  - pymatsolver>=0.2
-  - matplotlib
+  - pymatsolver-base>=0.2
+  - matplotlib-base
```

Comment:
make these dependent upon base version of the package.
* `matplotlib-base` avoids downloading the back QT package. This is also what would be installed if you did `conda install simpeg`.
* `pymatsolver-base`  makes it platform independent. `pymatsolver` on `conda` explicitly includes `pydiso` as a requirement due to legacy reasons which is not available on all platforms.